common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+{
+	"title": "Buttons",
+	"language": "en",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-09"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#purpose">Purpose</a></li>
+	<li><a href="#basic">Basic use</a></li>
+	<li><a href="#enhanced">Enhanced use</a>
+	<ul>
```

Comment:
Put its inner examples at the same level of "Basic use"

Reply:
Addressed in commit  "Button: Addressed Review Comments"